src/odemis/gui/util/test/img_test.py

```diff
@@ -1311,17 +1311,14 @@ def test_image_on_ovv(self):
         ovv_im = model.DataArray(numpy.zeros((11, 11, 3), dtype=numpy.uint8))
         ovv_im.metadata[model.MD_PIXEL_SIZE] = (1, 1)
         ovv_im.metadata[model.MD_POS] = (0, 0)
-#         tile = model.DataArray(255 * numpy.ones((3, 3, 3), dtype=numpy.uint8))
-        tile = model.DataArray(numpy.zeros((5, 5, 3), dtype=numpy.uint8))
-        tile[1:4, 1:4, :] = 255
-        tile.metadata[model.MD_POS] = (-3.5, 3.5)
-        # tile.metadata[model.MD_POS] = (0, 0)
-        tile.metadata[model.MD_PIXEL_SIZE] = (1, 1)  # 1,1
+        tile = model.DataArray(255 * numpy.ones((3, 3, 3), dtype=numpy.uint8))
+        tile.metadata[model.MD_POS] = (-3, 3)
+        tile.metadata[model.MD_PIXEL_SIZE] = (1, 1)
         ovv_im_new = insert_tile_to_image(tile, ovv_im)
-        print(ovv_im_new[:, :, 0])  # DEBUG
+        # print(ovv_im_new[:, :, 0])  # DEBUG
         # rectangle with edges (-4,4), (-2,4), (-4,2), (-4,2) should be white now
         # (6,6) is the center, so this corresponds to a rectangle with top left at (2,2) (=(1,1) when counting from 0)
-        numpy.testing.assert_array_equal(ovv_im_new[1:4, 1:4, :], tile)
+        numpy.testing.assert_array_equal(ovv_im_new[1:4, 1:4, :], 255)
```

Comment:
shouldn't the second argument be an iterable?

Reply:
Well, I've just found out that not for this numpy test. It actually does something like `a == b`, and in numpy it's allowed that a and b are not the same dimension (in which case there some kind "dimension unfolding").